Sources/xdgTestHelper/main.swift

```diff
@@ -19,7 +19,7 @@ import FoundationNetworking
 #endif
 #if os(Windows)
 import WinSDK
-#elseif os(Android)
+#elseif canImport(Android)
```

Comment:
I think that this should still be an `os` check.

Reply:
The two are pretty much equivalent, but I use this to guard imports like this and `os(Android)` for genuine OS differences that don't involve importing these C functions. I have seen this done for other platforms in the Swift codebase, so I'm following that distinction for Android too.

Reply:
This is only done for Darwin platforms because there is a large set of them and we don't have a way to uniformly identify them at once.